### ISSUE TITLE
Middle-clicking on tab causes browser window to jump to next display 

### ISSUE THREAD

**Issue by Brave-Matt** [brave/brave-browser#3113]
## Description 

_Note: I remember seeing this issue reported previously around the time Beta was released but can't find any issues/documentation on it._

Seeing several reports of users with multiple monitors/displays claiming that  middle-clicking (or mouse-wheel click) on a tab causes the browser window in focus to jump to the next display rather than closing the tab (intended effect).

This seems to only effect Windows users so far but other OS may also exhibit this behavior (more testing required). I personally was unable to reproduce this when testing all Brave builds on my dual-monitor setup using Windows 10.

## Steps to Reproduce

   1. Launch Brave on a device connected to at least 1 (active) additional display
   2. Open 1+ tabs and visit any site
   3. Middle-click on one of the tabs to close out the tab

## Actual result:

Browser window jumps from current display (where clicked) to the next display. 

## Expected result:
Middle-click should close the tab as if it were closed/x'd out normally.

## Reproduces how often: 

Easily/consistent **for the users who are experiencing the issue**. I'm unable to reproduce the issue on my end using a similar setup.

## Brave version (brave://version info)

Release: v0.58.21
Beta: v0.59.28
Dev: v0.60.17
Note: I also had each user (willing) attempt to install any of the other builds to see if it was isolated to a specific one - it was not. Issue persists across all Brave releases.
### Reproducible on current release:
- Does it reproduce on brave-browser dev/beta builds? 
Yes

### Additional Information

Last Note: Users say that performing this action in Chrome does _not_ reproduce the issue. Chrome closes the tab as intended.
Main (active) thread, 3 different users reporting:
https://community.brave.com/t/browser-window-jumps-to-next-display-when-closing-a-tab/40441/18

**Comment by fuse314** [comment]
Additional info: When "middle-clicking" on the very bottom part of the tab (green area in image), the tab closes as expected, when "middle-clicking" on the rest of the tab (red area in image), the window jumps to the next display.
![image](https://user-images.githubusercontent.com/927896/52793920-1991cf00-306f-11e9-8c2e-90410c7e1000.png)

**Comment by Brave-Matt** [comment]
@fuse314 this is great info! Appreciate it!

**Comment by Brave-Matt** [comment]
+1's from Community - Both these users are reporting that they have DisplayFusion installed on Windows 10 machines. may or may not be relevant.
https://community.brave.com/t/new-install-keeps-switching-monitors/44694/3

**Comment by landshrk** [comment]
Yep! Same issue here! I made [a post about it](https://community.brave.com/t/new-install-keeps-switching-monitors/44694/2) over in the Brave forums, and one of your wonderful helpdesk folks  (Mattches) linked this. Another user (Daemonic) mentioned that if you head into DisplayFusion's settings, go to the Window Management tab, and turn off 'Enable window caption bar middle-clicking to move windows to the next monitor', it would solve the problem. Or at least offer a workaround.

Tried it, and it works. Not ideal, for sure, but a solid workaround.

I'm currently writing to Binary Fortress to make sure they're aware of what's going on.

Thanks for all the help, and for making such a cool browser!

**Comment by landshrk** [comment]
Hey, so like I said above, I reached out to Binary Fortress. They just responded with some good news. Apparently, you folks have made them well aware of the problem, and they're working on a fix scheduled to come out with the next update. So we have that to look forward to.

**Comment by Spysix** [comment]
Display Fusion does have that middle clicking to move windows behavior. But on chrome, middle clicking on a tab to close will close the tab and _not_ move the window. Same as on Waterfox.

What I found interesting though, is the behavior does not apply to all windows evenly. It would affect slack, but not Chrome or Waterfox. Seems like it's just ideal to disable that feature in displayfusion.

**Comment by Brave-Matt** [comment]
+1 from reddit:
https://www.reddit.com/r/brave_browser/comments/bhb6ml/i_just_installed_brave_and_a_weird_bug_just/

**Comment by Brave-Matt** [comment]
+1 from reddit:
https://www.reddit.com/r/brave_browser/comments/bkufww/middle_clicking_on_a_tab_moves_the_window_to/

**Comment by Brave-Matt** [comment]
Update:
Just spoke to Binary Fortress support regarding this issue:
![image](https://user-images.githubusercontent.com/3299098/57412990-849efd80-71a7-11e9-9800-2f03b15b3e39.png)

**Comment by bsclifton** [comment]
@Brave-Matt is this definitively linked to Display Fusion? (if so let's update the issue title)

**Comment by Brave-Matt** [comment]
@Clifton every user I've spoken to/seen with this issue has had Display Fusion installed as well to manage multiple displays. 

This doesn't account for reports I havent seen obviously but the link seems very likely. BF team also acknowledges that the issue seems to be on their end. 

Let me know how I can further confirm the link between the two.

**Comment by puviyarasan** [comment]
I have DisplayLink installed and I kept hitting this issue till I saw Brave-Matt's workaround above, it worked thanks.

**Comment by Brave-Matt** [comment]
+1 https://community.brave.com/t/empty-section-of-tab-bar-doesnt-act-as-title-bar-like-other-apps/549022/2
